tests/furusiya/conftest.py

```diff
@@ -0,0 +1,15 @@
+from attrdict import AttrDict
+import pytest
+import config as prod_config
+import os
+
+# Has to be done before other imports
+path = os.path.join('furusiya', 'config.json')       
+with open(path, 'rt') as f:
+    raw_json = f.read()
+
+prod_config.load(raw_json)

```

Comment:
There is already a fix :/
https://github.com/nightblade9/furusiya/blob/ae8ad850c5fd4931679bcfbe0878110e20d9019e/tests/conftest.py#L10-L13

Good job on making it use a fixture, though I don't see how useful that would be. We're not actually using config *inside* tests; rather, it is imported in the classes we're testing. So the current fix would be sufficient.

Reply:
The current fix doesn't work.

I initially stumbled on the same fix, and everything looked great. Then I created a second test class *just to see what happens*, and guess what? It started failing with the usual `dict doesn't have attributes` value.

Thinking through my Python course material, I realized that this code *is only executed once when it's imported the first time,* whether manually with `import` statements or automatically by `PyTest`.

Turning it into a fixture guarantees that we have access to the one-and-only correctly-evaluated instance. So, yeah, we do need this ^-^

Reply:
Fair enough. Merged!